Sources/TuistCore/Cache/CacheCategory.swift

```diff
@@ -25,7 +25,7 @@ public enum CacheCategory: String, CaseIterable, RawRepresentable {
         case .builds:
             return "BuildCache"
         case .tests:
-            return "TestsCache"
+            return "incremental-tests"
```

Comment:
```suggestion
            return "IncrementalTestsCache"
```

Maybe better to keep the same format for the directory name as for the others here?

Reply:
> Maybe better to keep the same format for the directory name as for the others here?

I suggest changing that in Tuist 4 to eliminate some redundancy in the naming. For example, `~/.tuist/cache/BuildCache` could be `~/.tuist/cache/binaries`. In fact, I'm thinking about taking it a bit further and aligning our path with the [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html), so the path would be `$XDG_CACHE_HOME/tuist/binaries`. The list of directories would end up being:

```
$XDG_CACHE_HOME/binaries
$XDG_CACHE_HOME/selective-tests
$XDG_CACHE_HOME/project-description-helpers
$XDG_CACHE_HOME/manifests
```